content/vi/docs/reference/glossary/annotation.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Annotation
```

Comment:
I'd expect this to be localized, possibly with the original term shown as well. If this is the normal term that a Vietnamese person would use in technical writing, it's of course fine to use that.